packages/create/package.json

```diff
@@ -2,7 +2,6 @@
   "name": "@sku-lib/create",
   "version": "0.0.0",
   "description": "Create sku projects",
-  "private": true,
```

Comment:
Removing to avoid changsets from erroring. See https://github.com/seek-oss/sku/pull/1378